core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/messaginglayer/MessagingManager.scala

```diff
@@ -0,0 +1,37 @@
+package edu.uci.ics.amber.engine.architecture.messaginglayer
+
+import akka.actor.ActorRef
+import edu.uci.ics.amber.engine.architecture.receivesemantics.FIFOAccessPort
+import edu.uci.ics.amber.engine.common.ambermessage.WorkerMessage.DataMessage
+import edu.uci.ics.amber.engine.common.ambermessage.WorkflowMessage
+import edu.uci.ics.amber.engine.common.tuple.ITuple
+
+class MessagingManager(val fifoEnforcer: FIFOAccessPort) {
```

Comment:
This is not a real MessgingManager, it only works as a wrapper for the FIFOAccessPort. It should also take care of sending messages.

Reply:
I think we can continue with the name. I will be adding sending in later PRs.